corehq/apps/dump_reload/sql/dump.py

```diff
@@ -32,23 +32,45 @@
     UniqueFilteredModelIteratorBuilder('scheduling.SMSContent', SimpleFilter('alertevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.SMSContent', SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.SMSContent',
+                                       SimpleFilter('randomtimedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.SMSContent',
+                                       SimpleFilter('casepropertytimedevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.EmailContent', SimpleFilter('alertevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.EmailContent', SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.EmailContent',
+                                       SimpleFilter('randomtimedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.EmailContent',
+                                       SimpleFilter('casepropertytimedevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.SMSSurveyContent',
                                        SimpleFilter('alertevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.SMSSurveyContent',
                                        SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.SMSSurveyContent',
+                                       SimpleFilter('randomtimedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.SMSSurveyContent',
+                                       SimpleFilter('casepropertytimedevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.IVRSurveyContent',
                                        SimpleFilter('alertevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.IVRSurveyContent',
                                        SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.IVRSurveyContent',
+                                       SimpleFilter('randomtimedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.IVRSurveyContent',
+                                       SimpleFilter('casepropertytimedevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.CustomContent', SimpleFilter('alertevent__schedule__domain')),
     UniqueFilteredModelIteratorBuilder('scheduling.CustomContent', SimpleFilter('timedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.CustomContent',
+                                       SimpleFilter('randomtimedevent__schedule__domain')),
+    UniqueFilteredModelIteratorBuilder('scheduling.CustomContent',
+                                       SimpleFilter('casepropetytimedevent__schedule__domain')),
```

Comment:
typo, looks like this is what's breaking tests

Reply:
thanks, updated